### ISSUE TITLE
[bug]: v5.2.0 - gallery stops populating with new images.

### ISSUE THREAD

**Issue by tortoise74** [invoke-ai/InvokeAI#7212]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Linux

### GPU vendor

Nvidia (CUDA)

### GPU model

3090

### GPU VRAM

24Gb

### Version number

5.2.0

### Browser

131.0.3 (64-bit)

### Python dependencies

{
  "accelerate": "0.30.1",
  "compel": "2.0.2",
  "cuda": "12.4",
  "diffusers": "0.27.2",
  "numpy": "1.26.4",
  "opencv": "4.9.0.80",
  "onnx": "1.16.1",
  "pillow": "11.0.0",
  "python": "3.10.15",
  "torch": "2.4.1+cu124",
  "torchvision": "0.19.1+cu124",
  "transformers": "4.41.1",
  "xformers": null
}

Just noticed xformers is not installed that may explain a few other issues perhaps?

### What happened

The gallery has stopped populating with new images. Though the images and thumbnails are still be generated correctly and can be browsed from the output directory.
Even after shutting down and restarting the application this still happens for new images.
I set the log to debug but nothing shows.

### What you expected to happen

The gallery should be populated.
There should be an error message if it cannot be for any reason.

### How to reproduce the problem

Generated many images over many cycles without deleting any with a queue of different models.
There were some errors. Workflows do not work for me. I also have invokeai-web on a loop in a script as it sometimes dies due to lack of main memory (particularly if something else running uses main memory).

### Additional context

I am running on Ubuntu 24.04 under WSL on windows 11.  (my linux box does not have a good enough GPU)

I suspect if I clear out some images it will come back but I can leave it in its current broken state for a while in case there is something to look for that explains what's going wrong.

**Comment by psychedelicious** [comment]
Do you have this set to `Send to Gallery`?
![image](https://github.com/user-attachments/assets/0a97b869-e49c-45bf-b975-3c4bf6e707cd)

If it is instead set to `Send to Canvas`, your generations will be on the canvas instead of the gallery until you accept them.

**Comment by tortoise74** [comment]
Yes. Definitely set to send to gallery. I ended up resetting the web UI and clearing intermediates and it works now.
However, the images I generated before doing that are still missing from the gallery (but present in the output directory).

**Comment by psychedelicious** [comment]
Ok, glad it's working. You could technically drag those images back in but there's no automated way to restore them at this time.

Hmm, curious why it happened. I haven't run into this before. One thing to note - when you click Invoke, the queued generations all have the `Send to...` setting baked in. So, changing it after queuing generations, will not re-route the images.
